### DISCUSSION TITLE
[Bug]: 开启了自动战斗但会被点击取消

### DISCUSSION THREAD

**Discussion by Forevercyj** [moesnow/March7thAssistant#66] [Q&A] [answered]
### 请确保您已阅读以上注意事项，并勾选下方的确认框。

- [X] 我已经使用一个干净且无其它非必要的插件的环境测试过，问题依旧存在。
- [X] 我已经在 [Issue Tracker](https://github.com/moesnow/March7thAssistant/issues) 中找过我要提出的问题，没有找到相同问题的ISSUE。
- [X] 我已知晓并同意，若这个 Issue 是关于其他非程序本身问题，则我的 Issue 可能会被无条件自动关闭/锁定。
- [X] 我已知晓并同意，如果我不按照模板提供的指示进行填写，则我的 Issue 可能会被无条件删除。

### March7thAssistant 版本 （或 Commit Id）

1.6.9

### 重现步骤

之前不知道某种情况下弹出的命令窗让我选是否自动战斗，我当时选了“否”（第三个选项是“我啷个晓得”）。

### 期望的结果是什么？

能自动战斗

### 实际的结果是什么？

在游戏设置中已经启用了连续自动战斗，但每次开启游戏后，自动战斗的按钮会被点击，即被取消了自动战斗。

### 补充说明（可选）

在游戏设置中已经启用了连续自动战斗，但每次开启游戏后，自动战斗的按钮会被点击，即被取消了自动战斗。
原因估计是之前不知道某种情况下弹出的命令窗让我选是否自动战斗，我当时选了“否”（第三个选项是“我啷个晓得”）。但至今不知道这个命令该如何调出或设置。

谢谢！

**Comment by moesnow** [comment, marked as answer]
修改 `3rdparty\Fhoe-Rail\config.json` 或删除后重新配置
